### ISSUE TITLE
Stray '.' in README.md

### ISSUE THREAD

**Issue by scribe** [djkoloski/rkyv#171]
The example code in README.md appears to have a stray `.` character.

```Rust
serializer.serialize_value(&value).unwrap().
```

I suspect it should be
```Rust
serializer.serialize_value(&value).unwrap();
```

**Comment by djkoloski** [comment]
Good eye! I probably retyped this instead of copying from the tests. I'll fix this up.

**Comment by djkoloski** [comment]
Fixed by 90adf1fa2a668f83162d8b8e3b5407a6009b927a
